app/src/main/java/com/infomaniak/mail/utils/Extensions.kt

```diff
@@ -150,6 +150,12 @@ fun Date.isLastWeek(): Boolean {
 //endregion
 
 //region UI
+fun Fragment.isInPortrait(): Boolean = requireContext().isInPortrait()
+fun Context.isInPortrait(): Boolean = !isInLandscape()
+
+fun Fragment.isInLandscape(): Boolean = requireContext().isInLandscape()
+fun Context.isInLandscape(): Boolean = resources.getBoolean(R.bool.isInLandscape)
```

Comment:
why not using `resources.configuration.orientation == ORIENTATION_LANDSCAPE` or portrait instead of creating 2 res files